### PR TITLE
Bug 1673911 - Remove more data from treeherder-prototype2

### PR DESCRIPTION
**Pull request by ionutgoldan**
**Bug link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1673911

### CODE REVIEW

File: tests/model/cycle_data/test_treeherder_cycling.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    assert JobType.objects.filter(id__in=original_job_type_ids).count() == len(
+        original_job_type_ids
+    )
+    assert JobGroup.objects.filter(id__in=original_job_group_ids).count() == len(
+        original_job_group_ids
+    )
+    assert Machine.objects.filter(id__in=original_machine_ids).count() == len(original_machine_ids)
+
+
+def test_cycle_job_with_performance_data(
```
Comment (esanuandra) [inline tests/model/cycle_data/test_treeherder_cycling.py:163]: What do you think about adding a comment here explaining why we need to also test with `performance data` in `test_treeherder_cycling.py`?

File: treeherder/model/management/commands/cycle_data.py [hunk 2]
```diff
@@ -89,16 +103,24 @@ def prune(id_name, model):
...
         sleep_time: int,
         is_debug: bool = None,
+        days: int = None,
+        max_runtime: timedelta = None,
         strategies: List[RemovalStrategy] = None,
         **kwargs,
     ):
         super().__init__(chunk_size, sleep_time, is_debug)
-        self.strategies = strategies or RemovalStrategy.fabricate_all_strategies(chunk_size)
+        self.max_runtime = max_runtime or PerfherderCycler.DEFAULT_MAX_RUNTIME
```
Comment (beatrice-acasandrei) [inline treeherder/model/management/commands/cycle_data.py:120]: Here you can use the new class `MaxRuntime`

Comment (ionutgoldan) [reply treeherder/model/management/commands/cycle_data.py:120]: Good catch! 🎣

**Review by beatrice-acasandrei — APPROVED** [review]

File: treeherder/model/management/commands/cycle_data.py [hunk 3]
```diff
@@ -531,6 +578,17 @@ def add_arguments(self, parser):
...
         )
+        parser.add_argument(
+            '--days',
+            action='store',
+            dest='days',
+            type=int,
+            help=(
+                f'Data cycle interval expressed in days. '
+                f'Minimum {MINIMUM_PERFHERDER_EXPIRE_INTERVAL} days '
+                f'when expiring performance data.'  # unless on prototype environment
```
Comment (sarah-clements) [inline treeherder/model/management/commands/cycle_data.py:589]: Do you mean "unless on prototype2 environment"?

**Review by sarah-clements — APPROVED** [review]
lgtm

File: treeherder/model/management/commands/cycle_data.py (hunk 3 above)
Comment (ionutgoldan) [reply treeherder/model/management/commands/cycle_data.py:589]: Yes, you are right. I'm reformulating this, to avoid confusion.

**Pull request merged**